node/src/tests/resharing.rs

```diff
@@ -270,8 +283,8 @@ async fn test_key_resharing_multistage() {
 
 /// Test that signatures during resharing
 /// are also processed.
+#[serial] // this test relies on metrics for timing
```

Comment:
@netrome this one, unfortunately, I could not get rid off. The rest can be removed, as there are no race conditions.